doc/writing.md

```diff
@@ -150,14 +148,14 @@ Hello. ++I have added this part++
 
 ### Line break
 
-You can force line breaks in text (carriage return) using `<br>`, which is the
+You can force line breaks in text (carriage return) using `<br>`, similar to
```

Comment:
Well, it seemed a bit out of place to me. If you suggest, would revert it to the original version here.